hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java

```diff
@@ -98,18 +114,44 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
      * @param ledgerConfig
      * @return the information about the file
      */
-    private @Nullable Optional<FileInfo> infoForFile(
+    private Optional<FileInfo> infoForFile(
             @NonNull final FileID fileID,
             @NonNull final ReadableFileStore fileStore,
-            @NonNull final LedgerConfig ledgerConfig) {
-        final var meta = fileStore.getFileMetadata(fileID);
+            @NonNull final LedgerConfig ledgerConfig,
+            @NonNull final ReadableUpgradeStoreImpl upgradeStore,
+            @NonNull final FilesConfig fileServiceConfig)
+            throws IOException {
+
+        FileMetadata meta = null;
+        long contentSize = 0L;
+        // the update file always will be for the node, not a particular ledger that's why we just compare the num
```

Comment:
Nit:
upgrade is for the entire network, not a node.  It's across shards and realms, however, which is why we ignore the shard and realm values.

Reply:
fixed the comment. thank you